stats_test.go

```diff
@@ -68,7 +88,11 @@ func TestUDSExportError(t *testing.T) {
 }
 
 func TestNilAggregation(t *testing.T) {
-	exporter := testExporter(Options{})
+	exporter, err := testExporter(Options{})
+	if err != nil {
+		t.Error(err)
+	}
+
```

Comment:
Also here, no need for this line. The closing curly already provides enough spacing.